plugins/repository-gcs/qa/google-cloud-storage/build.gradle

```diff
@@ -29,6 +30,10 @@ import static org.elasticsearch.gradle.PropertyNormalization.IGNORE_VALUE
 apply plugin: 'elasticsearch.standalone-rest-test'
 apply plugin: 'elasticsearch.rest-test'
 
+dependencies {
+    testCompile project(path: ':plugins:repository-gcs')
```

Comment:
We should add a TODO here asking to:
* Flatten out the structure of this project so we only have a single root module build like we have for S3